test/print_test.cpp

```diff
@@ -0,0 +1,58 @@
+#include <vector>
+
+#if COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION == 2
+#include <geometry_msgs/msg/point.hpp>
+#elif COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION == 1
+#include <geometry_msgs/Point.h>
+#else
+#error "Undefined or unknown COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION"
+#endif
+
+#include <common_robotics_utilities/print.hpp>
+#include <common_robotics_utilities/voxel_grid.hpp>
+
+#include <gtest/gtest.h>
+
+namespace
+{
+namespace test
+{
+struct DummyType {
```

Comment:
minor Opening brace should be on the following line to match style.

Reply:
See https://github.com/calderpg/common_robotics_utilities/pull/14/commits/e816a2bfbe3ba312d2c29fe01f3b8b6b8816699a.